### PR TITLE
Activate autocomplete for docker images

### LINKED ISSUE #278: Autocomplete docker images

### PR DESCRIPTION
**Pull request by kashizui**
choices parameter to `add_argument` will automatically be picked up by argcomplete.

Fixes #278 

@percyliang please review

### CODE REVIEW

File: codalab/bundles/run_bundle.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class RunBundle(NamedBundle):
     METADATA_SPECS = list(NamedBundle.METADATA_SPECS)
     # Note that these are strings, which need to be parsed
     # Request a machine with this much resources and don't let run exceed these resources
-    METADATA_SPECS.append(MetadataSpec('request_docker_image', basestring, 'Which docker image (e.g., codalab/ubuntu:1.9) we wish to use.'))
+    METADATA_SPECS.append(MetadataSpec('request_docker_image', basestring, 'Which docker image (e.g., codalab/ubuntu:1.9) we wish to use.', choices=("codalab/ubuntu", "codalab/tensorflow-cuda7.0-352.39")))
```
Comment (percyliang) [inline codalab/bundles/run_bundle.py]: 'choices' somehow suggests that this is a closed set of options, but it's not in this case - people can request any docker image they want...is this the place we want to add it?

I thought we would want a DockerImageCompleter, with hard-coding for now, but later something that calls 'docker search'.

Comment (kashizui) [inline codalab/bundles/run_bundle.py]: That makes sense

**Comment by kashizui** [comment]
Used the new approach

@percyliang please review

**Comment by percyliang** [comment]
LGTM!

**Pull request merged**